ap_api/callbacks/customer/index.mdx

```diff
@@ -5,16 +5,19 @@ order: 0
 
 import {EndpointsTable} from "@site/src/components/EndpointsTable";
 
+Customer callbacks are designed to work with customer data. When working with the Anchor Platform, businesses will store KYC data defined by [SEP-12].
+
+However, because the Anchor Platform does not store this data, it must fetch it from the business server. This set of endpoints allows the Anchor Platform to forward data stored on the business side to the wallet in SEP format.
```

Comment:
Lets start with the most important piece, which is that we don't store KYC data, which is why we need these endpoints.

```suggestion
The Anchor Platform does not store customer KYC data. Instead, the Anchor Platform towards this data to and requests it from the business. The customer callbacks are designed for this purpose. Currently, requests & responses for these endpoints are almost identical to the [SEP-12 KYC API specification][sep-12].
```